api/v1/routes/blog.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.orm import Session
+from typing import List
+
+from api.v1.models.blog import Blog
+from api.v1.schemas.blog import BlogResponse
+from api.db.database import get_db
+
+blog = APIRouter(tags=["Blog"])
+
+@blog.get("/blogs", response_model=List[BlogResponse])
```

Comment:
add the `/blogs` `prefix` to the `APIRouter`  then the get route can be `/`, so everyone using this router can have `/blogs` by default

Reply:
But the other endpoints has prefix blog instead of blogs

Reply:
@Dev-wonderful The change have been made